modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/SpringCodegen.java

```diff
@@ -498,7 +498,9 @@ public void preprocessOpenAPI(OpenAPI openAPI) {
                             List<Map<String, String>> tags = new ArrayList<Map<String, String>>();
                             for (String tag : operation.getTags()) {
                                 Map<String, String> value = new HashMap<String, String>();
-                                value.put("tag", tag);
+                                // Replace single backslashes with double backslash to prevent 
+                                // String escape compilation errors
+                                value.put("tag", tag.replaceAll("(?<!\\\\)\\\\(?!\\\\)", "\\\\\\\\"));
```

Comment:
@hebra thanks for the fix. What about using the `sanitizeTag` method instead?

Ref: https://github.com/OpenAPITools/openapi-generator/blob/2c051f265c522d73c60b5310f3858d7e1fb60b82/modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractJavaCodegen.java#L1385